web/templates/stream_settings/first_stream_created_modal.hbs

```diff
@@ -0,0 +1,34 @@
+<div class="micromodal" id="first_stream_created_modal" aria-hidden="true">
+    <div class="modal__overlay" tabindex="-1">
+        <div class="modal__container" role="dialog" aria-modal="true"
+          aria-labelledby="first_stream_created_modal_label">
+            <header class="modal__header">
+                <h1 class="modal__title" id="first_stream_created_modal_label">
+                    <span>
+                        {{#tr}}
+                        Stream <b>#{stream_name}</b> created!
+                        {{/tr}}
+                    </span>
+                </h1>
+                <button class="modal__close" aria-label="{{t 'Close modal' }}" data-micromodal-close></button>
+            </header>
+            <main class="modal__content">
+                <span>
+                    {{t 'You will now see the stream you created. To go back to stream settings, you can:' }}
+                    <ul>
+                        <li>
+                            {{#tr}} Click on {{/tr}}
+                            <b>{{> ../inline_decorated_stream_name stream=stream}} </b>
+                            {{#tr}} at the top of your zulip window.{{/tr}}
```

Comment:
This i18n approach doesn't work; different languages have different word orders, os you can never cut up a sentence in different translation blocks this way. See `git grep z-stream web/templates/` for examples of doing this correctly.

The whitespace here is also weird; generally one doesn't want extra spaces like this.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html.

Reply:
Understood.